### PR TITLE
Official meteor package

### PR DESCRIPTION
**Pull request by marceloschmidt**
This pull requests adds emojione:emojione as official meteor package for Emojione.

### CODE REVIEW

**Comment by engelgabriel** [comment]
:+1:

**Comment by mikebe11** [comment]
Thanks a lot for taking the time to put that together and submit it. We'll update the readme. Will there be any additional instructions needed for installation or just 
meteor add qnub:emojione     changed to    meteor add emojione:emojione

**Comment by engelgabriel** [comment]
There is one extra part of the process, we must help us with. If you go to Meteor.com and register yourself, and create an organization called "emojione" and add us as members, we can set it all up for you. Than I'll talk to the MDG to add the repository on the auto-build list of packages, so any time you tag a version on github, it updates the meteor version database too.

**Comment by mikebe11** [comment]
@engelgabriel  I created the account but the organization name is already taken.

**Comment by engelgabriel** [comment]
What username did you use for your own account?
They use the same unique index key as the organization names.

**Comment by mikebe11** [comment]
username: emojione

**Comment by engelgabriel** [comment]
Thats the problem... you may have to ask them to change your personal username to something else... so you can create a organization with that name.

It happened to me too..  :(

Whats we did, was to create my personal account with other username, and asked them to turn my company account into an organization.

**Comment by engelgabriel** [comment]
See https://github.com/MeteorCommunity/discussions/issues/14

File: package.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Package.describe({
+  name: 'emojione:emojione',
+  summary: 'Meteor Package of http://www.emojione.com/ set',
+  version: '1.4.1',
+  git: 'https://github.com/Ranks/emojione.git',
+  name: "emojione:emojione"
```
Comment (splendido) [inline package.js]: duplicate line

**Comment by engelgabriel** [comment]
So, did you get the account name sorted?

**Comment by mikebe11** [comment]
I emailed the folks at Meteor.com on Friday about altering the username on the account I created. I'm just waiting to hear back.

**Comment by mikebe11** [comment]
@engelgabriel  I heard from Meteor over two weeks ago and replied with the necessary changes. I followed up a week ago and they said he'll get to it as soon as he can.
Just giving you a heads up. Haven't forgotten about the package. Still waiting for the account fix.

**Comment by engelgabriel** [comment]
I'll chase them! :)

**Comment by mikebe11** [comment]
@engelgabriel  It took a long time but the account issues should be squared away now. I have a Meteor account under my name with organization: emojione  Are you still looking to assist with creating the official Meteor package? If so, let me know what you need from me.

**Comment by engelgabriel** [comment]
COOL! FINALLY!

I am in San Francisco this week, presenting our https://rocket.chat project at the Meteor DevShop

I'll get back to this when I return to my office, is that ok?

**Comment by mikebe11** [comment]
Of course, that's fine. Thanks.

**Pull request merged**